exp-player/addon/components/exp-player.js

```diff
@@ -58,7 +58,10 @@ export default Ember.Component.extend(FullScreen, {
                 expData: this.get('expData'),
                 sequence: sequence
             };
-            this.sendAction('saveHandler', payload);  // call the passed-in action with payload
+            var exitUrl = this.get('experiment.exitUrl');
+            this.sendAction('saveHandler', payload,  () => {
```

Comment:
It's a callback function. Looks like the corresponding code from https://github.com/CenterForOpenScience/experimenter/pull/55/files was lost during merge.  

Will update to a list before merge.